client/src/modules/CourseManagement/components/CourseEventModal/index.tsx

```diff
@@ -151,12 +151,16 @@ export function CourseEventModal({ data, onCancel, courseId, onSubmit }: Props)
           <Form.Item
             name="dateTime"
             label="Date and Time"
-            rules={[{ required: true, message: 'Please enter date and time' }]}
+            rules={[{ required: true, message: 'Please enter start date and time' }]}
           >
             <DatePicker format="YYYY-MM-DD HH:mm" showTime={{ format: 'HH:mm' }} />
           </Form.Item>
         </Col>
-        <Form.Item name="endTime" label="End Date and Time">
+        <Form.Item
+          name="endTime"
+          label="End Date and Time"
+          rules={[{ required: true, message: 'Please enter start date and time' }]}
+        >
```

Comment:
copy-paste? is it supposed to be required? if so message should be: `Please enter end date and time`

Reply:
fixed, thank you!